### DISCUSSION TITLE
OPUS Codec Supported?

### DISCUSSION THREAD

**Discussion by renekuehl** [music-assistant/support#855] [Feature requests and ideas]
As a lot of my Audio Files dont show up in Music Assistant (.opus) I wonder if playing them through Music Assistant is supported? I can't find any Description of which Formats are supported. Thank you in Advance.

**Comment by marcelveldt** [comment]
I've added a bunch of file formats but forgot about opus. So this I snot yet supported but it's a very easy fix for me to add more file formats. Maybe someone can give me a list of some more I should care about and I'll add them all at once, no problem.

These are currently supported:

```
TRACK_EXTENSIONS = ("mp3", "m4a", "mp4", "flac", "wav", "ogg", "aiff", "wma", "dsf")
PLAYLIST_EXTENSIONS = ("m3u", "pls")
```

**Reply by renekuehl** [reply]
thank you for clearing this up ;)

**Reply by gomerust** [reply]
Hi, i was wondering why opus would still be not supported. Is it just not configured, or are there some other Problems with this format?

**Reply by OzGav** [reply]
Probably just that no one answered Marcel’s request for a list so it fell by the wayside

**Reply by gomerust** [reply]
then i would kindly request opus to be added to the list. ;-)

**Reply by OzGav** [reply]
Please raise an issue

**Comment by marcelveldt** [comment]
Opus support is now added, available in next release (beta86)

**Reply by jmberg** [reply]
Can you please also add it to AUDIOBOOK_EXTENSIONS? Seems very simple so I can also open a PR if that's preferred.

**Reply by OzGav** [reply]
Yes open a PR

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
